python/ray/parameter.py

```diff
@@ -252,6 +257,20 @@ def update_if_absent(self, **kwargs):
         self._check_usage()
 
     def _check_usage(self):
+        if self.worker_port_list is not None:
+            for port_str in self.worker_port_list.split(","):
+                try:
+                    port = int(port_str)
+                except ValueError as e:
+                    raise ValueError(
+                        "worker_port_list must be a comma-separated " +
+                        "list of integers: {}".format(e))
```

Comment:
```
In [3]:                 try:
   ...:                     port = int(port_str)
   ...:                 except ValueError as e:
   ...:                     raise ValueError(
   ...:                         "worker_port_list must be a comma-separated " +
   ...:                         "list of integers: {}".format(e))
   ...:
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-3-160cfabed35e> in <module>
      1 try:
----> 2     port = int(port_str)
      3 except ValueError as e:

ValueError: invalid literal for int() with base 10: 'hi'

During handling of the above exception, another exception occurred:

ValueError                                Traceback (most recent call last)
<ipython-input-3-160cfabed35e> in <module>
      4     raise ValueError(
      5         "worker_port_list must be a comma-separated " +
----> 6         "list of integers: {}".format(e))
      7

ValueError: worker_port_list must be a comma-separated list of integers: invalid literal for int() with base 10: 'hi'
```

Reply:
To avoid unnecessary exception chaining, maybe do `raise ValueError(...) from None`

Reply:
wow, I had no idea you could do this! very cool